app.js

```diff
@@ -16,13 +16,75 @@ function getBooks() {
 }
 
 function displayBooks(data) {
-  let bookDiv = "";
+  let bookDiv = '';
   data.forEach((book) => {
     const title = `<a class = 'title' href= "books/book.html?id=${book.id}">  ${book.title}</a>  `;
-
     const author = `<div class = 'author'> ${book.author.first_name} ${book.author.last_name}</div> `;
     const bookCover = `<img class = 'book-cover' src = ${book.cover_url} >`;
     bookDiv += `<div class='book'> ${title} ${author} ${bookCover}   </div> `;
   });
   allBooks.innerHTML = bookDiv;
 }
+
+// Login Form popup
+document.getElementById('btn-Form').addEventListener('click', openUserForm);
+document
+  .getElementById('popup-close-btn')
+  .addEventListener('click', closeUserForm);
+document
+  .getElementById('popup-close-x')
+  .addEventListener('click', closeUserForm);
+
+function openUserForm() {
+  document.getElementById('form-popup').style.display = 'block';
+}
+function closeUserForm() {
+  document.getElementById('form-popup').style.display = 'none';
+}
+
+////////////////////////////////////////////////////////////////
+//                        Login system
+////////////////////////////////////////////////////////////////
+
+// Validating the entered data and show User Name on top
+
+document.getElementById('btn-submit').addEventListener('click', validate);
+
+function validate(e) {
+  e.preventDefault();
+  // ELEMENTS variables for username and password
+  let userName = document.getElementById('uname');
+  let passWord = document.getElementById('psw');
```

Comment:
userName is not correct. 
First, this variable is a reference to the html element and not the actual value of the username. 
Second, it's spelled `username `and not `userName`, but this should be called something like `usernameInput`

The same goes fot the password element

Reply:
and use full names for ids and not abbreviations. So not psw and uname

Reply:
The same comment goes for the other variables too. Change them accordingly, please

Reply:
alright got you and updating right now